go/oasis-test-runner/scenario/e2e/runtime.go

```diff
@@ -557,6 +557,12 @@ func (sc *runtimeImpl) initialEpochTransitions() error {
 			return fmt.Errorf("failed to wait for nodes: %w", err)
 		}
 
+		// Temporary....
+		// TODO: with the change that keymanager initially registers without
+		// status, before it is initialized, we should wait here that keymanager
+		// is initialized. Add a method to wait for KM initialization?
+		time.Sleep(10 * time.Second)
```

Comment:
If the "register as a keymanager on the first status update" fix is ok (see changelog fragment), i'll fix this properly by waiting for KM initialization, but didn't want to implement that before the fix is acked, as it might not be needed if we decide for a different fix (e.g. maybe just retrying the initialization)

Reply:
Wouldn't something be needed in either case with the "key manager always registers"? E.g., even if you retry, the retry could happen later than the key manager actually registering with correct `ExtraInfo`? So maybe we should do #2130 and then you could call `WaitReady` on the key manager node(s) to check if they are really ready?

Reply:
Yeah would be, but if i revert the "km always registers" and only replace it it with retrying the initialization no changes are needed here - so i think i'll do just that.

Reply:
Once we have #2130 we should change the code here to wait for ready, and we can also bring back the "km always registers" fix, since i think is generally a bit nicer than the current "hack" to only register after initial failure :-)

Reply:
Maybe mention this in #2130 (and add a comment in the code) so that we don't forget about it?

Reply:
done, i'll mention on the issue once this is merged so i can permalink to code